rpc/client/rpc_test.go

```diff
@@ -25,8 +25,8 @@ import (
 )
 
 func getHTTPClient() *client.HTTP {
-	rpcAddr := rpctest.GetConfig().RPC.ListenAddress
-	return client.NewHTTP(rpcAddr, "/websocket")
+	rpcAddr := strings.Replace(rpctest.GetConfig().RPC.ListenAddress, "tcp", "http", -1)
```

Comment:
we should use the default constructor here

Reply:
Ah, sorry, I misunderstood.